tutorials/issue1235.ipynb

```diff
@@ -0,0 +1,494 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.    print(flux_to_l_value)
</pre><p>Perhaps add a "Fluxes projected using MortarProjection" to the print?</p><p>And "Fluxes projected by hard-coded indices" below</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pmgbergen/porepy/pull/1286/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>